demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
```

Comment:
Is this the right language to clarify?

```suggestion
1. Download the English language file `cht-core/config/default/translations/messages-en.properties` from the default config
```

Reply:
This would be the file from your instance. Likely would end up in your downloads folder.